core/src/main/java/org/apache/carbondata/core/util/CarbonUtil.java

```diff
@@ -2070,6 +1994,20 @@ public static void dropDatabaseDirectory(String dbName, String storePath)
     }
   }
 
+  public static DataType getDataType(char type) {
```

Comment:
Suggest to move all conversion to DataType enum, including this one and `ColumnPageEncoderMeta.convertType`. And also move `CarbonCommonConstants.BIG_INT_MEASURE` and related constants to DataType enum also.

Reply:
ok